cmd/utils.go

```diff
@@ -1045,3 +1045,22 @@ func checkTime() {
 		}
 	}
 }
+
+func setNewIndexURL() {
```

Comment:
Please add some comments to this function to describe its purpose. Especially in this case, I'm looking for something that says this is temporary code and should be removed in the future.